### PR TITLE
Add OCLC icon

### LINKED ISSUE #12183: Request: OCLC
### Brand Name

OCLC

### Website

https://www.oclc.org

### Popularity Metric

Similarweb Global Rank: [3,737](https://www.similarweb.com/website/oclc.org)

### Official Resources for Icon and Color

- SVG: https://www.oclc.org/content/dam/oclc/design-images/navigation-logo.svg

### Additional Comments

Media kit available: https://www.oclc.org/en/news/media-kit.html (only EPS; same as #11514)

### PR DESCRIPTION
**Pull request by timmy471**
oclc

**Issue:** closes #12183

**Popularity metric:** 3,737

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Review by LitoMore — CHANGES_REQUESTED** [review]
I can see some differences. Please don't stretch the icon while resizing.

**Comment by timmy471** [comment]
@LitoMore update the Icon, please review

File: _data/simple-icons.json [hunk 1]
```diff
@@ -18548,4 +18554,4 @@
             "source": "https://www.zyte.com"
         }
     ]
-}
+}
```
Comment (LitoMore) [inline _data/simple-icons.json:18557]: Please don't edit other lines. Can you revert this change?

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (LitoMore) [reply _data/simple-icons.json]: I just updated this for you.

**Pull request merged**